v2/common/headers.avsc

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "record",
+  "name": "Header",
+  "namespace":"io.renku.events.v2",
```

Comment:
Since the header carries the schema version (which really applies to the payloads), maybe we should not version the header then? Otherwise when reading messages, we have to guess-decode the header first, trying v1 and v2 etc. Wdyt?

Reply:
I moved the v2 header into `header` directory so it is not versioned